src/lib.rs

```diff
@@ -752,4 +752,7 @@ pub use value::*;
 #[cfg(feature = "derive")]
 pub use sea_query_derive::Iden;
 
+#[cfg(feature = "attr")]
```

Comment:
I don't see you adding this future to `Cargo.toml`, can you add?

Reply:
> 

![image](https://user-images.githubusercontent.com/45243386/163838040-14e3b104-cfe5-4d2d-afac-8171c2667d0f.png)
Already added

Reply:
Oh sorry, I missed.